### PR TITLE
added Missing Data section to the vignette

### PR DESCRIPTION
**Pull request by KKulma**
added Missing Data section to the vignette

## Description
added a section on Missing Data, described  MissingDataProvider() & make_missing() and gave examples on how to generate data of different types.

## Related Issue
#85 

<!--- Did you remember to include tests? Unless you're just changing
No tests required

### CODE REVIEW

File: vignettes/charlatan.Rmd [hunk 1]
```diff
@@ -42,7 +42,7 @@ sequence data
 
 ## Contributing
 
-See the **Contributing to charlatan** vignette
+See the [**Contributing to charlatan**](https://github.com/ropensci/charlatan/blob/master/vignettes/contributing.Rmd) vignette
```
Comment (sckott) [inline vignettes/charlatan.Rmd]: can you link to https://docs.ropensci.org/charlatan/articles/contributing.html instead

File: vignettes/charlatan.Rmd [hunk 2]
```diff
@@ -169,6 +169,33 @@ ch_credit_card_security_code()
 ch_credit_card_security_code(10)
 ```
 
+## Missing data 
+`charlatan` makes it very easy to generate fake data with missing entries. First, you need to run `MissingDataProvider()` and then make an appropriate `make_missing()` call specifying the data type to be generated. This method picks a random number (`N`) of slots in the input the `make_missing` vector and then picks `N` random positions that will be replaced with NA matching the input class.
```
Comment (sckott) [inline vignettes/charlatan.Rmd]: > of slots in the input the `make_missing` vector

"of slots in the input `make_missing` vector"

**Comment by sckott** [comment]
thanks! just a few comments

File: vignettes/charlatan.Rmd (hunk 1 above)
Comment (KKulma) [reply vignettes/charlatan.Rmd]: now done in 4810d9ee7a0cf3359eb07d12c7410501abbe2cb2

File: vignettes/charlatan.Rmd (hunk 2 above)
Comment (KKulma) [reply vignettes/charlatan.Rmd]: now done in d3adc9972ea7850d2789070d844033d743cf53a6

**Pull request merged**